### PR TITLE
Antonella1

### PR DESCRIPTION
**Pull request by antobrita**

### CODE REVIEW

File: 4001-5000/LIT4872MaqdemaWangel.xml [hunk 1]
```diff
@@ -52,31 +52,92 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
+              <label>Preface</label>
+              <div type="textpart" subtype="title" xml:id="TitleChapter1"><ab>ግብር፡ ቀዳማዊ፡</ab></div>
+              <div type="textpart" subtype="incipit"><ab>ወግብሩሰ፡ ለዝንቱ፡ መጽሓፍ፡ ማሕየዊ፡ ከመ፡ ይርባሕ፡ ቦቱ፡ ሕይወት፡
+              ዘለዓለም፡ ለደቂቀ፡ እጓለ፡ እመሕያው። ወበጺሖቶሙ፡ ኀበ፡ ፈጣሪሆሙ፡ በቃል፡ ወበምግባር።</ab></div>
+              <div type="textpart" subtype="explicit"><ab>ወበተግሣጻቲሁ፡ ወበተካይዶቱ፡ ወበነጊሮቱ፡ ረሰዮሙ፡ ለደቂቀ፡ እጓለ፡
+              እመሕያው፡ ልጹቃን፡ ኀበ፡ ፈጣሪሆሙ፡ ይግበሩ፡ ምግባራቲሁ፡ ወይረሱ፡ መንግሥቶ።</ab></div>
+            </div>
+            <div type="textpart" subtype="section" xml:lang="gez" xml:id="Chapter2">
+              <label>Preface</label>
+              <div type="textpart" subtype="title" xml:id="TitleChapter2"><ab>ወካልኡ፡ ብቍዔት፡</ab></div>
```
Comment (MassimoVilla) [inline 4001-5000/LIT4872MaqdemaWangel.xml:100]: በቍዔት፡ instead of ብቍዔት፡

File: 4001-5000/LIT4872MaqdemaWangel.xml [hunk 2]
```diff
@@ -52,31 +52,92 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
-                <div type="textpart" subtype="chapter" xml:lang="gez" xml:id="Chapter1"/>
-                <div type="textpart" subtype="chapter" xml:lang="gez" xml:id="Chapter2"/>
-                <div type="textpart" subtype="chapter" xml:lang="gez" xml:id="Chapter3"/>
-                <div type="textpart" subtype="chapter" xml:lang="gez" xml:id="Chapter4"/>
-                <div type="textpart" subtype="chapter" xml:lang="gez" xml:id="Chapter5"/>
-                <div type="textpart" subtype="chapter" xml:lang="gez" xml:id="Chapter6"/>
-                <div type="textpart" subtype="chapter" xml:lang="gez" xml:id="Chapter7"/>
+        <div type="edition">
+            <note>The following division in textual units is based on
+              <bibl><ptr target="bm:Cowley1977NT"/><citedRange unit="page">147-160</citedRange></bibl>and on
```
Comment (MassimoVilla) [inline 4001-5000/LIT4872MaqdemaWangel.xml:75]: please leave a space before "and": `</bibl> and`

File: 4001-5000/LIT4872MaqdemaWangel.xml [hunk 3]
```diff
@@ -52,31 +52,92 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
+              <div type="textpart" subtype="title" xml:id="TitleChapter1"><ab>ግብር፡ ቀዳማዊ፡</ab></div>
+              <div type="textpart" subtype="incipit"><ab>ወግብሩሰ፡ ለዝንቱ፡ መጽሓፍ፡ ማሕየዊ፡ ከመ፡ ይርባሕ፡ ቦቱ፡ ሕይወት፡
+              ዘለዓለም፡ ለደቂቀ፡ እጓለ፡ እመሕያው። ወበጺሖቶሙ፡ ኀበ፡ ፈጣሪሆሙ፡ በቃል፡ ወበምግባር።</ab></div>
+              <div type="textpart" subtype="explicit"><ab>ወበተግሣጻቲሁ፡ ወበተካይዶቱ፡ ወበነጊሮቱ፡ ረሰዮሙ፡ ለደቂቀ፡ እጓለ፡
+              እመሕያው፡ ልጹቃን፡ ኀበ፡ ፈጣሪሆሙ፡ ይግበሩ፡ ምግባራቲሁ፡ ወይረሱ፡ መንግሥቶ።</ab></div>
+            </div>
+            <div type="textpart" subtype="section" xml:lang="gez" xml:id="Chapter2">
+              <label>Preface</label>
+              <div type="textpart" subtype="title" xml:id="TitleChapter2"><ab>ወካልኡ፡ ብቍዔት፡</ab></div>
+              <div type="textpart" subtype="incipit"><ab>ወበቍዔተ፡ ዝንቱ፡ መጽሓፍ፡ ብጹሕ፡ ካዕበ፡ እስመ፡ ውእቱ፡ ዘይትንበብ፡
```
Comment (MassimoVilla) [inline 4001-5000/LIT4872MaqdemaWangel.xml:101]: ይትነበብ፡ ?

File: 4001-5000/LIT4872MaqdemaWangel.xml [hunk 4]
```diff
@@ -52,31 +52,92 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
+            <div type="textpart" subtype="section" xml:lang="gez" xml:id="Chapter3">
+              <label>Preface</label>
+              <div type="textpart" subtype="title" xml:id="TitleChapter3"><ab>ወሣልስ፡ ሥርዓት፡</ab></div>
+              <div type="textpart" subtype="incipit"><ab>ወሥርዓተ፡ ዝንቱ፡ መጽሐፍ፡ ክቡር፡ ወይእቲኬ፡ አንብቦቱ፡ ኵሎ፡ ጊዜ፡
+              ወኀዲገ፡ ሕሊናት፡ ለዓቂበ፡ አቅማሪሁ፡ ወተአምራቲሁ፡ ወትእኅዘ፡ ቦቱ።</ab></div>
+              <div type="textpart" subtype="explicit"><ab>ወናሁ፡ ተጠየቀት፡ ሥርዓት፡ ወጸንዓት፡ ንብረቱ።</ab></div>
+            </div>
+            <div type="textpart" subtype="section" xml:lang="gez" xml:id="Chapter4">
+              <label>Preface</label>
+              <div type="textpart" subtype="title" xml:id="TitleChapter4"><ab>ወራቡዕ፡ ትእምርት፡</ab></div>
```
Comment (MassimoVilla) [inline 4001-5000/LIT4872MaqdemaWangel.xml:115]: perhaps ወራ**ብ**ዕ፡?

File: 4001-5000/LIT4872MaqdemaWangel.xml [hunk 5]
```diff
@@ -52,31 +52,92 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
+            </div>
+            <div type="textpart" subtype="section" xml:lang="gez" xml:id="Chapter6">
+              <label>Preface</label>
+              <div type="textpart" subtype="title" xml:id="TitleChapter6"><ab>ወሳድስ፡ ምስማክ፡</ab></div>
+              <div type="textpart" subtype="incipit"><ab>ወውእቱ፡ ለአይ፡ ግብር፡ ይሴኒ ወለኣይ፡ ገጽ፡ መክፈልተ፡ ጥበብ፡ ዘይትፈቀድ።</ab></div>
+              <div type="textpart" subtype="explicit"><ab>ወናሁ፡ ተበየነ፡ ምስማከ፡ ዝንቱ፡ መጽሓፍ፡ ወለአይ፡ ግብር፡ ይኄይስ።</ab></div>
+            </div>
+            <div type="textpart" subtype="section" xml:lang="gez" xml:id="Chapter7">
+              <label>Preface</label>
+              <div type="textpart" subtype="title" xml:id="TitleChapter7"><ab>ወሳበዕ፡ አርእስተ፡ ነገር፡</ab></div>
```
Comment (MassimoVilla) [inline 4001-5000/LIT4872MaqdemaWangel.xml:134]: ወሳብዕ፡

File: 4001-5000/LIT4872MaqdemaWangel.xml [hunk 6]
```diff
@@ -52,31 +52,92 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
-                <div type="textpart" subtype="chapter" xml:lang="gez" xml:id="Chapter3"/>
-                <div type="textpart" subtype="chapter" xml:lang="gez" xml:id="Chapter4"/>
-                <div type="textpart" subtype="chapter" xml:lang="gez" xml:id="Chapter5"/>
-                <div type="textpart" subtype="chapter" xml:lang="gez" xml:id="Chapter6"/>
-                <div type="textpart" subtype="chapter" xml:lang="gez" xml:id="Chapter7"/>
+        <div type="edition">
+            <note>The following division in textual units is based on
+              <bibl><ptr target="bm:Cowley1977NT"/><citedRange unit="page">147-160</citedRange></bibl>and on
+              <bibl><ptr target="bm:Zuurmond1989EvangeliumMarc"/><citedRange unit="page">14-15</citedRange></bibl>
+              while the incipit and explicit displayed are from
```
Comment (MassimoVilla) [inline 4001-5000/LIT4872MaqdemaWangel.xml:77]: Perhaps "incipits and explicits"? I would leave out "displayed" (it seems a bit redundant to me), but it's my personal opinion.

**Review by MassimoVilla — CHANGES_REQUESTED** [review]
Thank you Anto for your work! :)

File: 4001-5000/LIT4872MaqdemaWangel.xml (hunk 1 above)
Comment (antobrita) [reply 4001-5000/LIT4872MaqdemaWangel.xml:100]: Typo... thank you!

File: 4001-5000/LIT4872MaqdemaWangel.xml (hunk 2 above)
Comment (antobrita) [reply 4001-5000/LIT4872MaqdemaWangel.xml:75]: ok

File: 4001-5000/LIT4872MaqdemaWangel.xml (hunk 3 above)
Comment (antobrita) [reply 4001-5000/LIT4872MaqdemaWangel.xml:101]: yes

File: 4001-5000/LIT4872MaqdemaWangel.xml (hunk 5 above)
Comment (antobrita) [reply 4001-5000/LIT4872MaqdemaWangel.xml:134]: thank you for correcting all these typos, Massimo

**Review by SusanneHummel — APPROVED** [review]

File: 4001-5000/LIT4872MaqdemaWangel.xml (hunk 6 above)
Comment (antobrita) [reply 4001-5000/LIT4872MaqdemaWangel.xml:77]: alright, thank you

**Comment by SusanneHummel** [comment]
Congratulation to your first PR!!

**Review by MassimoVilla — APPROVED** [review]
very good!

**Pull request merged**